homeassistant/components/waze_travel_time/config_flow.py

```diff
@@ -97,25 +102,36 @@ def async_get_options_flow(
 
     async def async_step_user(self, user_input=None):
         """Handle the initial step."""
+        errors = {}
         if user_input is not None:
-            await self.async_set_unique_id(
-                slugify(
-                    f"{DOMAIN}_{user_input[CONF_ORIGIN]}_{user_input[CONF_DESTINATION]}"
-                )
-            )
-            self._abort_if_unique_id_configured()
-            return self.async_create_entry(
-                title=(
-                    user_input.get(
-                        CONF_NAME,
-                        (
-                            f"{DEFAULT_NAME}: {user_input[CONF_ORIGIN]} -> "
-                            f"{user_input[CONF_DESTINATION]}"
-                        ),
+            if not await self.hass.async_add_executor_job(
```

Comment:
We can invert this check and move the error update to below the return for create entry.